### PR TITLE
FUSETOOLS2-611 - fixing flaky tests

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

File: src/test/command.start.test.ts [hunk 1]
```diff
@@ -283,8 +269,8 @@ testUtils.BROWSER_TYPES.forEach(function (browserConfig) {
 
 function checkContainsAtlasMapTitle() {
 	const expectedAtlasMapTitle = '<title>AtlasMap Data Mapper UI</title>';
-	waitUntil(() => atlasMapWebView.default.currentPanel._panel.webview.html.includes(expectedAtlasMapTitle));
+	waitUntil(() => atlasMapWebView.default?.currentPanel?._panel.webview?.html?.includes(expectedAtlasMapTitle));
 	expect(
-		atlasMapWebView.default.currentPanel._panel.webview.html,
+		atlasMapWebView.default?.currentPanel?._panel?.webview?.html,
```
Comment (apupier) [inline src/test/command.start.test.ts:274]: the failign assertion is ignored. you can notice it by changing the expectedAtlasmapTitle to somethign that it no t part of the page. The tests are still passing

**Review by apupier — CHANGES_REQUESTED** [review]

File: src/test/command.start.test.ts (hunk 1 above)
Comment (lhein) [reply src/test/command.start.test.ts:274]: Did u test that? When I change `expectedAtlasMapTitle` to something else the tests fail just fine for me.

Comment (apupier) [reply src/test/command.start.test.ts:274]: :'-( code is not compiled automatically

it is failing for me too

**Review by bfitzpat — APPROVED** [review]
definitely more straightforward

**Review by djelinek — APPROVED** [review]
👍

**Review by apupier — APPROVED** [review]

**Pull request merged**